src/var-scope.md

```diff
@@ -1,6 +1,6 @@
-# Reduce Scope of Variables
+# Reduce Scope of Variables and Constants
```

Comment:
This might mess up existing links?

If so, we can just update description and examples

Reply:
Yeah let's not edit the heading if possible. I think the description is pretty clear.